aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Orchestrator.java

```diff
@@ -35,6 +37,8 @@
  * and {@link AppSync}.
  */
 public final class Orchestrator {
+    private static final Logger LOG = Amplify.Logging.forNamespace("amplify:aws-datastore");
```

Comment:
Assume you'll update this prior to merging, just a flag to remind you.

Reply:
To the `forCategory` factory? Yea, I did the two PRs as siblings, I'll have to update whichever one goes in second.